packages/ppx/test/snapshot/reason/media-queries-and-selectors.t/run.t

```diff
@@ -964,7 +964,7 @@
       CssJs.style([|
         CssJs.label("SelectorsMediaQueries"),
         CssJs.media(
-          {js|(min-width: 600px)|js},
+          {js|(min-width: 600px) |js},
```

Comment:
Why this space is added?

Reply:
it's a bug causing the same in other places too.